neurons/validator.py

```diff
@@ -658,7 +653,7 @@ async def run_step(self):
 
         # Compute wins and win rates per uid.
         wins, win_rate = pt.validation.compute_wins(
-            uids, losses_per_uid, batches_to_use, uid_to_block
+            uids, losses_per_uid, batches_new, uid_to_block
```

Comment:
batches_to_use? len(batches_new) != len(batches_old)

Reply:
Again, this was the case originally in v3.1.4, we introduced `batches_to_use` in our release instead, but now I am reverting it, for the exact same reason as my comment about `losses` above.